--- a/jscoq/js/coq-manager.js	2016-11-21 11:00:16.242296619 +0100
+++ b/jscoq/js/coq-manager.js	2016-08-24 12:24:53.085729507 +0200
@@ -170,10 +170,8 @@
             wrapper_id: 'ide-wrapper',
             base_path:  "./",
             init_pkgs: ['init'],
-            all_pkgs:  ['init', 'math-comp', 'mtac',
-                        'coq-base', 'coq-arith', 'coq-reals',
-                        'coquelicot', 'flocq', 'tlc', 'sf', 'cpdt', 'color', 'relalg', 'unimath',
-                        'plugin-utils', 'extlib', 'mirrorcore']
+            all_pkgs:  ['init', 'math-comp',
+                        'coq-base', 'coq-arith', 'coq-reals' ]
         };
 
         this.options = copyOptions(options, this.options);
